src/VCAuthn/IdentityServer/Endpoints/AuthorizationEndpoint/AuthorizeEndpoint.cs

```diff
@@ -53,18 +54,25 @@ ILogger<AuthorizeEndpoint> logger
         
         public async Task<IEndpointResult> ProcessAsync(HttpContext context)
         {
-            if (!HttpMethods.IsPost(context.Request.Method))
-            {
-                return new StatusCodeResult(HttpStatusCode.MethodNotAllowed);
-            }
-            
-            if (!context.Request.HasFormContentType)
-            {
-                return new StatusCodeResult(HttpStatusCode.UnsupportedMediaType);
+            _logger.LogDebug("Processing Authorize request");
+
+            NameValueCollection values;
+            switch (context.Request.Method)
+            {
+                case "GET":
```

Comment:
Is there an enum or const available to use here?

Reply:
The one that is built into the framework uses `static strings` and cannot be used in switches. Ended up using strings here instead of re-inventing enums.